stream-video-android-compose/src/main/kotlin/io/getstream/video/android/compose/ui/components/call/renderer/internal/LandscapeVideoRenderer.kt

```diff
@@ -293,6 +297,33 @@ internal fun BoxScope.LandscapeVideoRenderer(
                 }
             }
         }
+
+        else -> {
+            BoxWithConstraints(modifier = Modifier.fillMaxHeight()) {
+
+                LazyVerticalGrid(
+                    modifier = Modifier.fillMaxSize(),
+                    columns = GridCells.Fixed(3),
+                    content = {
+                        items(callParticipants.size) { key ->
+                            // make 2 items exactly fit available height
+                            val itemHeight = with(LocalDensity.current) {
+                                (constraints.maxHeight / 2).toDp()
+                            }
+                            val participant = callParticipants[key]
+                            videoRenderer.invoke(
+                                modifier = modifier.height(itemHeight),
+                                call = call,
+                                participant = participant,
+                                style = style.copy(
+                                    isFocused = dominantSpeaker?.sessionId == participant.sessionId
+                                )
+                            )
```

Comment:
We can improve here by using items (list + key) like the example below:

```kotlin
items(
    items = callParticipants,
    key = { it.sessionId }
) { participant ->
    // make 2 items exactly fit available height
    val itemHeight = with(LocalDensity.current) {
        (constraints.maxHeight / 2).toDp()
    }
    videoRenderer.invoke(
        modifier = modifier.height(itemHeight),
        call = call,
        participant = participant,
        style = style.copy(
            isFocused = dominantSpeaker?.sessionId == participant.sessionId
        )
    )
}
```

Providing a stable key enables item state to be consistent across data-set changes.